lessons/module-1/js-array-prototype-methods.md

```diff
@@ -6,41 +6,76 @@ tags: javascript, arrays, prototypes, mutator, accessor
 
 ## Learning Goals
 
-* Read documentation and explore array prototype methods
+* Summarize various array prototype methods
+* Use array prototype methods
 * Determine if a given method is a mutator or accessor
-* Build a process for self-guided learning, and determine which parts are most helpful for _you_
+* Identify learning strategies that are most helpful for <em>you</em>
 
 ## Vocabulary
 
 - `Array Prototype` - methods that are built specifically for arrays. They help us change the arrays themselves or get certain information out of them.
 - `Mutator` - methods that mutate, or change, the original array
 - `Accessor` - methods that do not mutate the original array, rather just give us some information about the array
 
-## Warm Up
+<section class="call-to-action">
+### Warm Up
+
+<div class="answer">
+
+### Exploration
 
-In your notebook, jot down your answers to the following:
+In your notebook, brainstorm how you might complete the following challenges. Don't use any code. Instead, write down your thought process as a set of instructions; this is sometimes referred to as 'pseudocode'. 
 
 ```javascript
-var instructors = ["Will", "Scott", "Hannah", "Casey"];
+var instructors = ["Scott", "Heather", "Jeremiah"];
 ```
 
 For the array above,
-- How would you programmatically find the number of elements?
```

Comment:
I think we could include the .length prompt too!

Reply:
I ended up adding an extra bullet point so student can think through that and hopefully the 'extra' marker will indicate that it's a slightly different concept or at least leave room for pause. Hopefully that's a happy medium!

Reply:
Thanks for the explanation - I think this is a great way to differentiate concepts